server/command.go

```diff
@@ -484,3 +485,74 @@ func (p *Plugin) subscribeCommand(parameters []string, channelID string, config
 
 	return fmt.Sprintf("Successfully subscribed to %s.%s", fullPath, hookStatusMessage)
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	gitlabCommand := model.NewAutocompleteData("gitlab", "[command]", "Avaliable commands: connect, disconnect, todo, subscribe, unsubscribe, me, settings, webhook")
+
+	connect := model.NewAutocompleteData("connect", "", "Connect your GitLab account")
+	gitlabCommand.AddCommand(connect)
+
+	disconnect := model.NewAutocompleteData("disconnect", "", "disconnect your GitLab account")
+	gitlabCommand.AddCommand(disconnect)
+
+	todo := model.NewAutocompleteData("todo", "", "Get a list of unread messages and merge requests awaiting your review")
+	gitlabCommand.AddCommand(todo)
+
+	subscribe := model.NewAutocompleteData("subscribe", "[command]", "Available commands: list")
+	subscribeList := model.NewAutocompleteData("list", "", "List current channel subscritpions")
+	subscribe.AddCommand(subscribeList)
+
+	subscribeChannel := model.NewAutocompleteData("subscribe", "owner[/repo] [features]", "Subscribe the current channel to receive notifications from a project")
```

Comment:
We already have `subscribe` subcommand adding this will make autocomplete as `/gitlab subscribe subscribe..`

Reply:
There seems to be something buggy with the UI when two sets of ```AutocompleteData``` have triggers that start with the same word.

Reply:
Ahh makes sense. Yes there is no way autocomplete can differentiate, so we should not do that. I'd suggest to use `/gitlab subscriptions` to list them. @hanzei thoughts?
This will change the command itself not only the autocomplete part.

Reply:
Two alternative options to consider

1. Leave the commands the way they are and don't provide autocomplete for either of the subscribe parameter sets. This has the advantage of not changing any command names which could break user workflows.

2. Change ```/gitlab subscribe owner[/repo] [features]``` to ```/gitlab subscribe add owner[/repo] [features]``` This seems more consistent with the slashCommand subCommand action parameters convention then ```/gitlab subscriptions``` but is more verbose then @iomodo suggestion.

Option 2 is my current preference, @iomodo @hanzei what do you think?

Reply:
I quite like the ideas as it makes the command much clearer.

Reply:
Thanks for the feedback everyone, I'll modify the PR to use Option 2. ```/gitlab subscribe add```